### PR TITLE
Explain the layer rendering options in 3D tab

### LINKED ISSUE #4793: [FEATURE] Vector layers are loaded in 3D view in background
Original commit: https://github.com/qgis/QGIS/commit/ab887771410d44d68eb8ca3c7fa616097683ac9f by wonder-sk

We are now using chunked entities for vector layers, so they
are split into several chunks (tiles) and each gets prepared
in background thread separately, not blocking main thread.

This is an initial implementation - work in progress.

### PR DESCRIPTION
**Pull request by DelazJ**
Fixes #4793 and update the screenshot accordingly

### CODE REVIEW

File: docs/user_manual/working_with_vector/vector_properties.rst [hunk 1]
```diff
@@ -1897,8 +1897,36 @@ See :ref:`data_defined_labeling` for more information.
...
 |3d| The :guilabel:`3D View` tab provides settings for vector layers that should
-be depicted in the :ref:`3D Map view <label_3dmapview>` tool. There are settings
-available for point, line and polygon layers.
+be depicted in the :ref:`3D Map view <label_3dmapview>` tool.
+
+For better performance, data from vector layers are loaded in the background,
+using multithreading, and rendered in tiles whose size can be controlled from
+the :guilabel:`Layer rendering` section of the tab:
+
+* :guilabel:`Zoom levels count`: determines how deep will be the quadtree.
```
Comment (havatv) [inline docs/user_manual/working_with_vector/vector_properties.rst:1906]:
```suggestion
* :guilabel:`Zoom levels count`: determines how deep the quadtree will be.
```

File: docs/user_manual/working_with_vector/vector_properties.rst [hunk 2]
```diff
@@ -1897,8 +1897,36 @@ See :ref:`data_defined_labeling` for more information.
...
+  For example, one zoom level means there will be a single tile for the whole layer.
+  Three zoom levels means there will be 16 tiles at the leaf level (every extra
+  zoom level multiplies that by 4). Default level value is ``3`` and maximum is ``8``.
```
Comment (havatv) [inline docs/user_manual/working_with_vector/vector_properties.rst:1909]:
```suggestion
  zoom level multiplies that by 4). The default is ``3`` and the maximum is ``8``.
```

File: docs/user_manual/working_with_vector/vector_properties.rst [hunk 3]
```diff
@@ -1897,8 +1897,36 @@ See :ref:`data_defined_labeling` for more information.
...
+* |checkbox| :guilabel:`Show bounding boxes of tiles`: especially useful if
+  there are some issues with tiles not showing up when they should
```
Comment (havatv) [inline docs/user_manual/working_with_vector/vector_properties.rst:1911]:
```suggestion
  there are issues with tiles not showing up when they should
```

**Comment by DelazJ** [comment]
This change assumes that the layer rendering section should be shown when "no symbols" is in use (https://github.com/qgis/QGIS-Documentation/issues/4793#issuecomment-623706842)

**Pull request merged**